### ISSUE TITLE
Improve --help command with an example

### ISSUE THREAD

**Issue by vielmetti** [deoxxa/don#3]
Current help command:

```
Edwards-Air:~ emv$ don --help
usage: don --public_url=PUBLIC_URL [<flags>]

Really really small OStatus node.

Flags:
  --help                         Show context-sensitive help (also try
                                 --help-long and --help-man).
  --addr=":5000"                 Address to listen on.
  --database="don.db"            Where to put the SQLite database.
  --public_url=PUBLIC_URL        URL to use for callbacks etc.
  --log_level=INFO               How much to log.
  --pubsub_refresh_interval=15m  PubSub subscription refresh interval.

```

If I have an account on e.g. mastodon.social, what do I type here to make this work?

**Comment by zeroware** [comment]
This is a node implementation. 
I don't think you need an account anywhere prior to launching don.

**Comment by deoxxa** [comment]
@zeroware that's correct. You should just be able to boot it up (with a public URL) and start using it.

The real question is... what exactly do you _do_ with don? Currently the answer is "not much" - it's a read-only node that only implements about half of the OStatus protocol. You put a username in the search box, don looks up the metadata, fetches the user's posts, and subscribes to their updates. And that's it.

In time it'll do more, and there'll be more to configure. I do hope to keep it to a minimum though - I really like the idea of "download binary -> immediately get started".

**Comment by zeroware** [comment]
Yes, It will be really nice to have a small node implementation even if you don't get all the features.
Maybe you can write in your README that this is a read-only node that can be used as a Atom/Feed aggregator for the Ostatus network ?
